mne/viz/_3d.py

```diff
@@ -682,3 +683,79 @@ def plot_sparse_source_estimates(src, stcs, colors=None, linewidth=2,
     surface.actor.property.shading = True
 
     return surface
+
+
+def plot_dipoles(dipoles, fwd, subject, subjects_dir, bgcolor=(1, ) * 3,
```

Comment:
Also, I think in my example I simplified the plotting code a bit, you might want to re-build this based on what I used there. For example, you shouldn't need a forward solution to be able to do the plotting...